<launch>

    <param name="wheelRadius" value="0.069" />
    <param name="halfLenght" value="0.195" />
    <param name="halfWidth" value="0.164" />
    <param name="gearRatio" value="5" />
    <param name="tickRes" value="38" />
    <param name="use_sim_time" value="true" />
    <param name="msgInterval" value="1" />

    <node pkg="project_1" type="velocitiesTicks" name="ComputeVelocitiesTicks" output="screen" />
    <node pkg="project_1" type="odometry" name="ComputeOdometry" output="screen" />
    <node pkg="project_1" type="tfBroadcast" name="tfBroadcast" />
    <node pkg="project_1" type="wheelsVel" name="WheelsVelocities" />

</launch>